src/style/content/preferences/options.less

```diff
@@ -64,7 +64,7 @@
   label::before {
     width: 22px;
     height: 22px;
-    border: 1px solid var(--gray-80);
+    border: 2px solid var(--gray-80);
```

Comment:
I don't understand why we have so many hardcoded pixel values. Arent borders and spacings streamlined for the app and therefore should be variables?

Reply:
Ain't nobody got time to have a conversation with me about what to do with css variables and theming going forward. There's no variable for radio-borders, should I had one.